lib/rubocop/cop/style/guard_clause.rb

```diff
@@ -277,7 +277,10 @@ def accepted_form?(node, ending: false)
         end
 
         def trivial?(node)
-          node.branches.one? && !node.if_branch.if_type? && !node.if_branch.begin_type?
+          node.branches.one? &&
+            node.if_branch &&
+            !node.if_branch.if_type? &&
+            !node.if_branch.begin_type?
```

Comment:
Can you tweaks to the following?

```suggestion
          return false unless node.if_branch

          node.branches.one? && !node.if_branch.if_type? && !node.if_branch.begin_type?
```